### PR TITLE
Work

### PR DESCRIPTION
**Pull request by 1v4n4**
In this milestone, I made a Math Magician website with three different pages. I also made jest tests for this website

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import Calculate from '../logic/calculate';
+
+const Calculator = () => {
+  const [data, setData] = useState({ total: null, next: null, operation: null });
+
+  const handleClick = (name) => {
+    setData(Calculate(name, data));
+  };
+
+  // const { total, next } = data;
```
Comment (rindrajosia) [inline src/components/Calculator.js]: - [x] [REQUIRED] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.

File: src/__tests__/Calculate.test.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  it('multiplies two numbers', () => {
+    const BTN = '*';
+    const data = { total: '2', operation: '*', next: '3' };
+    const result = Calculate(BTN, data);
+    expect(result.total).toEqual('6');
+  });
+});
+
+describe('- button', () => {
+  it('subtracts two numbers', () => {
```
Comment (rindrajosia) [inline src/__tests__/Calculate.test.js]: - [x] [REQUIRED] Please make sure that the `test description `is correct. Here for example, you should have `to add two numbers`

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <header>
+    <div className="headlineDiv">
+      <h1>Math Magicians</h1>
+    </div>
+    <nav>
+      <a className="navLink" href="/">Home</a>
+      <p className="navLink">|</p>
+      <a className="navLink" href="/Calculator">Calculator</a>
+      <p className="navLink">|</p>
+      <a className="navLink" href="/Quote">Quote</a>
```
Comment (rindrajosia) [inline src/components/Header.js]: [OPTIONAL] It will be nice if you use `Link (react router) ` here instead of `a href`  as you are building an SPA

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] all functionalities should be tested using[ React Testing Library](https://testing-library.com/docs/react-testing-library/intro/), `@testing-library/react,` see the [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/testing/useReactTestingLibrary.png) from [Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106580952/).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--

**Review by mahmoud717 — APPROVED** [review]
### STATUS: APPROVED ✔️ ✔️ 

**hi @1v4n4**

**Great job with making all of the requirements. the project is approved, you can go ahead and merge this pull request**

**Pull request merged**